### PR TITLE
US038 EntityCalendarDeleteDraftRecordPermanentlyTest added

### PR DESCRIPTION
**Pull request by 010V**
Calendar Create, Delete Draft Record, Delete from Bin Permanently Test added

### CODE REVIEW

File: test/specs/entityCalendarDraftDelete.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        browser.maximizeWindow();
+        browser.url('/');
+        userLogin("tester18@tester.test", "yjwc8pFJ0U");
+        reset();
+    });
+
+    let newEventTitle = "Calendar Event";
+    let newEventDescription = "Event description.";
+    let intNumbers = "28";
+    let decimalNumbers = "20.21";
```
Comment (Burdishka) [inline test/specs/entityCalendarDraftDelete.js:16]: We have testData.json file for data. Move this to testData.json

File: test/specs/entityCalendarDraftDelete.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    it('01 - Should click on the Sidebar menu', () => {
+        const sidebarMenu = "//div[@class = 'logo d-flex']";
```
Comment (Burdishka) [inline test/specs/entityCalendarDraftDelete.js:19]: You don't need to create any variables or selectors in this JS tests file. For data we use testData.json, for selectors we use selectors.json, and also we have expected.json for expected result. So move everything in each file. Same in 23, 31, 34, 37, 40, 43, 46, 54, 59, 62, 67, 70, 75, 77, 78. And also check that in each of these files we have common where we collect data, selectors that can be used by every engineer.

File: test/specs/entityCalendarDraftDelete.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        deletePermanentlyButton.click();
+    });
+    it('09 - Should verify Recycle Bin is empty', () => {
+        const recycleBin = $("//a/i[contains (text(), 'delete_outline')]");
+        recycleBin.click();
+        const contentRecycleBin = $("//div[@class = 'card-body']").getText();
+        const expectedEmptyBinText = "Good job with housekeeping! Recycle bin is currently empty!";
+        expect(contentRecycleBin).to.equal(expectedEmptyBinText);
+    });
+});
```
Comment (Burdishka) [inline test/specs/entityCalendarDraftDelete.js:81]: add one more empty line 82

File: test/specs/entityCalendarDraftDelete.js (hunk 1 above)
Comment (010V) [reply test/specs/entityCalendarDraftDelete.js]: done

File: test/specs/entityCalendarDraftDelete.js (hunk 2 above)
Comment (010V) [reply test/specs/entityCalendarDraftDelete.js]: done

File: test/specs/entityCalendarDraftDelete.js (hunk 3 above)
Comment (010V) [reply test/specs/entityCalendarDraftDelete.js]: done

**Comment by Burdishka** [comment]
Ok, good but where are verifications? If "it" doesn't have a verification then it no sense to make it. I suggest collecting all statements in one "it" with at least one verification.

**Comment by 010V** [comment]
> Ok, good but where are verifications? If "it" doesn't have a verification then it no sense to make it. I suggest collecting all statements in one "it" with at least one verification.

I thought "it" is like pre-condition, if it passes and allows you to move to next step, then no "expect" needed. I though we only have one expect per test... ?
Refactored, now 3 "It", each with "expect"

TY

**Review by SamJulia — APPROVED** [review]

**Review by Burdishka — APPROVED** [review]

**Pull request merged**